reactor-netty-http/src/test/java/reactor/netty/http/client/HttpClientWithTomcatTest.java

```diff
@@ -333,31 +327,25 @@ void contentHeader() {
 		fixed.dispose();
 	}
 
-	static Stream<Arguments> testIssue2825Args() {
-		Supplier<Publisher<ByteBuf>> postMono = () -> Mono.just(Unpooled.wrappedBuffer(PAYLOAD));
-		Supplier<Publisher<ByteBuf>> postFlux = () -> Flux.just(Unpooled.wrappedBuffer(PAYLOAD));
-
-		return Stream.of(
-				Arguments.of(Named.of("postMono", postMono), Named.of("bytes", PAYLOAD.length)),
-				Arguments.of(Named.of("postFlux", postFlux), Named.of("bytes", PAYLOAD.length))
-		);
-	}
+	@Test
+	void testIssue2825() {
+		tomcat.setMaxSwallowSize(MAX_SWALLOW_SIZE);
 
-	@ParameterizedTest
-	@MethodSource("testIssue2825Args")
-	void testIssue2825(Supplier<Publisher<ByteBuf>> payload, long bytesToSend) {
 		AtomicReference<SocketAddress> serverAddress = new AtomicReference<>();
 		HttpClient client = HttpClient.create()
 				.port(getPort())
 				.wiretap(false)
 				.metrics(true, ClientMetricsRecorder::reset)
-				.doOnConnected(conn -> serverAddress.set(conn.address()));
+				.doOnConnected(conn -> {
+					conn.channel().config().setOption(ChannelOption.SO_SNDBUF, 4 * 1024);
```

Comment:
This can be done as a direct configuration on the level of `HttpClient` -> `HttpClient.option(...)`

Reply:
applied suggestion in https://github.com/reactor/reactor-netty/pull/2903/commits/27457f55c1b01baeccb52fdcdfff6268020570d8